samples/angular-todo/src/app/features/journey/form/form.component.ts

```diff
@@ -79,14 +94,35 @@ export class FormComponent implements OnInit {
     this.submittingForm = true;
 
     try {
+      const selectIdCallback = step?.callbacks.find(
+        (cb) => cb.payload.type === 'SelectIdPCallback',
+      ) as SelectIdPCallback;
+
+      if (selectIdCallback) {
+        const nameCallBacksInputs = step?.payload.callbacks.find(
+          (cb) => cb.type === 'NameCallback',
```

Comment:
same as above here. also, instead of `.input` we have a `getInputValue` method you can chain onto the end of this.

Reply:
done